packages/kbn-es/package.json

```diff
@@ -10,6 +10,8 @@
     "dedent": "^0.7.0",
     "elasticsearch": "^14.1.0",
     "execa": "^0.10.0",
+    "getopts": "^2.0.6",
```

Comment:
I recommend adding `kbn-es` to the list of packages checked by eslint for dependencies: https://github.com/elastic/kibana/blob/master/.eslintrc.js#L93-L108